tests/acceptance/features/bootstrap/OccContext.php

```diff
@@ -1119,6 +1134,34 @@ public function theAdminHasAddedRemovedTheApplicableUserForMountUsingTheOccComma
 		$this->theCommandShouldHaveBeenSuccessful();
 	}
 
+	/**
+	 * @When the administrator lists all the created local storage using occ command
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function userListsLocalStorageMountUsingTheOccCommand() {
+		$this->listLocalStorageMount();
+	}
+
+	/**
+	 * @Then the following local storage should exist
+	 *
+	 * @param TableNode $mountPoints
+	 *
+	 * @return void
+	 */
+	public function theFollowingLocalStoragesShouldExist(TableNode $mountPoints) {
+		$expectedLocalStorages = $mountPoints->getColumnsHash();
+		$commandOutput = \json_decode($this->featureContext->getStdOutOfOccCommand());
+		foreach ($commandOutput as $i) {
```

Comment:
```suggestion
		foreach ($commandOutput as $storageEntry) {
```
Use a better variable name than `$i`